actions/shared/helm_deploy/action.yml

```
@@ -0,0 +1,137 @@
+      Example: 'templates/consul-service/consul_clean_infra_passport.yml'
+    required: true
+
+  service_branch:
+    description: |
+      Branch in the service repository
+    required: true
+
+  pipeline_branch:
+    description: |
+      Branch in the qubership-test-pipelines repository
+    required: true
+    default: 'main'
+
+  service_name:
+    description: |
+      Name that Helm will use to install the service
+    required: true
+
+  repository_name:
+    description: |
+      Service repository name (without organization prefix)
+      Example: 'qubership-consul' for https://github.com/Netcracker/qubership-consul
+    required: true
+
+  path_to_chart:
+    description: |
+      Path to helm chart within service repository
+      Example: 'charts/helm/consul-service'
+    required: true
+
+  components:
+    description: |
+      List of components where Docker images should be replaced 
+      with versions from the current branch
+      Example: 'statusProvisioner,integrationTests'
+    required: true
+
+  namespace:
+    description: |
+      Kubernetes namespace for service installation
+    required: true
+
+runs:
+  using: 'composite'
+  steps:
+    - name: Checkout pipeline
+      uses: actions/checkout@v4
+      with:
+        ref: '${{inputs.pipeline_branch}}'
+        repository: 'Netcracker/qubership-test-pipelines'
+        path: 'qubership-test-pipelines'
+
+    - name: Checkout repo
+      uses: actions/checkout@v4
+      with:
+        ref: '${{inputs.service_branch}}'
+        repository: 'Netcracker/${{inputs.repository_name}}'
+        path: '${{inputs.repository_name}}'
+
+    - name: Create namespace
+      if: inputs.deploy_mode == 'install'
+      run: kubectl create namespace '${{inputs.namespace}}'
+      shell: bash
+
+    - name: Create restricted resources
+      if: inputs.restricted == 'true'  && inputs.deploy_mode == 'install'
+      uses: Netcracker/qubership-test-pipelines/actions/shared/create_restricted_resources@update_actions
+      with:
+        path_to_template: ${{inputs.path_to_template}}
+        service_branch: ${{inputs.service_branch}}
+        service_name: ${{inputs.service_name}}
+        repository_name: ${{inputs.repository_name}}
+        path_to_chart: ${{inputs.path_to_chart}}
+        namespace: ${{inputs.namespace}}
+
+    - name: Copy template
+      run: |
+        cp qubership-test-pipelines/${{inputs.path_to_template}} ${{inputs.repository_name}}/${{inputs.path_to_chart}}
+      shell: bash
+
+    - name: Update images
+      run: >- 
+        #UPDATE IMAGES
+        SET_NEW_IMAGES=$(python qubership-test-pipelines/python/update_images.py 
+        --service_branch='${{inputs.service_branch}}' 
+        --path_to_values='${{inputs.repository_name}}/${{inputs.path_to_chart}}/values.yaml'
+        --components='${{inputs.components}}') >> $GITHUB_ENV
+      shell: bash
+
+    - name: Install service with helm
```

Have you considered the following?

```suggestion
    - name: helm ${{inputs.deploy_mode}} ${{inputs.service_name}}
```

It would make CI logs easier to understand